test/tests/rackhd11/test_rackhd11_api_workflows.py

```diff
@@ -89,5 +89,83 @@ def test_api_11_workflows_put_ipmi(self):
+                        "implementsTask": "Task.Base.Rest",
+                        "options":{
+                            "url":"http://localhost:8080/api/1.1/nodes",
+                            "method":"GET"
+                        },
+                        "properties": {}       
+                    }
+                },
+                {
+                    "label" : "REST-call-2",
+                    "taskDefinition": {
+                        "friendlyName":"2nd REST call",
+                        "injectableName":"Task.Rest.GET",
+                        "implementsTask": "Task.Base.Rest",
+                        "options":{
+                            "url":"http://localhost:8080/api/1.1/nodes",
+                            "method":"GET"
+                            },
+                        "properties": {}
+                    }       
+                },
+                {
+                    "label" : "REST-call-bad",
+                    "taskDefinition": {
+                        "friendlyName":"bad REST call",
+                        "injectableName":"Task.Rest.GET",
+                        "implementsTask": "Task.Base.Rest",
+                        "options":{
+                            "url":"http://localhost:8080/api/never/happen",
+                            "method":"GET"
+                        },
+                        "properties": {}
+                    }       
+                },
+                {
+                    "label" : "REST-call-final",
+                    "taskDefinition": {
+                        "friendlyName":"final REST call",
+                        "injectableName":"Task.Rest.GET",
+                        "implementsTask": "Task.Base.Rest",
+                        "options":{
+                            "url":"http://localhost:8080/api/1.1/nodes",
+                            "method":"GET"
+                        },
+                        "properties": {}
+                    },
+                    "waitOn":{
+                        "anyOf":{
+                            "REST-call-good": "failed",
+                            "REST-call-bad": "failed"
+                        },
+                        "REST-call-2": "succeeded"
+                    }
+                }
+            ]
+        } 
+
+        # Assume previous test will make sure this will success
+        setup_return = fit_common.rackhdapi("/api/1.1/workflows", action="put", payload=data_payload)
+        
+        if not MON_NODES:
+            run_return = fit_common.rackhdapi("/api/1.1/nodes/{0}/workflows/?name=Graph.REST.call", action="post")['JSON']
+            for taskItr in run_return.tasks:
+                if run_return.tasks.taskItr.friendlyName == "final REST call":
+                    self.assertEqual(run_return.tasks.taskItr.state, "pending", "unreachable task REST-call-final state should be pending but get "+ run_return.tasks.taskItr);
```

Comment:
The test case need wait the graph finished, and check the graph state is succeed, otherwise, you never validate the `anyOf` functionality.

Also below assertion is also necessary to make sure your graph definition is correct:
1. Check task "REST-call-good" succeed.
2. Check task "REST-call-bad" failed.
3. Check task "REST-call-2" succeeded.